addons/friendly_tracker/functions/fnc_hasTrackerVehicle.sqf

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+/*
+ * Author: veteran29
+ * Function checks if vehicle has tracking capabilities
+ *
+ * Arguments:
+ * 0: The vehicle that will be checked if it has a tracking capabilities <OBJECT>
+ *
+ * Return Value:
+ * 0: Has tracker <BOOL>
+ *
+ * Example:
+ * [bob] call afft_friendly_tracker_fnc_hasTrackerVehicle
+ *
+ * Public: No
+ */
+
+params ["_vehicle"];
+
+if (!GVAR(GPSVehicle)) exitWith {false};
+
+getNumber (configOf _vehicle >> "enableGPS") == 1 // return
```

Comment:
Do we really want to limit this only to configured vehicles? No support for custom variable?

Reply:
Wasn't there a support to explicitly add certain object into tracker? It will have different color if I recall tho. 🤔 And also it would be shown when empty, not sure if it's necessary.

Reply:
Also I wonder if adding caching here would improve performance.

Reply:
Caching would be nice indeed.

Yep, there is support for adding tracker to objects but it's a bit different behaviour. But I guess we can leave it for now and maybe add this later if needed.

Reply:
I've implemented caching and added ability to override config with `afft_friendly_tracker_enabledGps` variable on vehicle.